src/main/java/ee/tuleva/onboarding/epis/mandate/details/MandateDetailsDeserializer.java

```diff
@@ -19,16 +19,23 @@ public Object findDeserializer(Annotated a) {
         }
       };
 
+  public static MandateType deserializeMandateTypeField(JsonNode mandateDetailsRoot)
```

Comment:
Does it need to be public? Can be private, for proper encapsualtion

Reply:
We looked into sharing, probably didn't finish it here

Reply:
Ah, i see we still share this code